src/MMLib.SwaggerForOcelot/MMLib.SwaggerForOcelot.csproj

```diff
@@ -23,10 +23,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Kros.Utils" Version="1.8.1" />
-    <PackageReference Include="Microsoft.Extensions.Http" Version="2.1.1" />
-    <PackageReference Include="Ocelot" Version="12.0.1" />
-    <PackageReference Include="Swashbuckle.AspNetCore.SwaggerUI" Version="4.0.1" />
+    <PackageReference Include="Kros.Utils" Version="1.10.1" />
+    <PackageReference Include="Microsoft.Extensions.Http" Version="3.1.0-preview3.19553.2" />
```

Comment:
In this library I want referencing stable versions of external packages. No _rc_, _preview_, _alpha_, ... It's necessary for you referencing this concrete version?

Reply:
Well i figured that visual studio downloaded alpha releases by mistake, so i just used the latest stable one now
Ocelot 13.8
Microsoft.Extensions.Http 3.0.1

for the Swashbuckle.AspNetCore.SwaggerUI it's been a long time since they released a stable release and since the whole OpenApi full support is with the version 5 i figured i can win you time until they do release a stable one.